apps/web/components/AppendedArticleComponents/AppendedArticleComponents.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import dynamic from 'next/dynamic'
+import { GetSingleArticleQuery } from '@island.is/web/graphql/schema'
+
+const IcelandicNamesSearcher = dynamic(
+  () => import('../IcelandicNamesSearcher/IcelandicNamesSearcher'),
+  {
+    ssr: false,
+  },
+)
+
+type Article = GetSingleArticleQuery['getSingleArticle']
+
+interface Props {
+  article: Article
+}
+
+export const AppendedArticleComponents = ({ article }: Props) => {
+  switch (article?.id) {
+    // Leit í mannanafnaskrá
```

Comment:
Keep comments in English
https://docs.devland.is/technical-overview/code-standards#language